goal_src/pc/progress-pc.gc

```diff
@@ -1302,12 +1306,12 @@
                                                             (lookup-text! *common-text* (game-text-id off) #f)
                                                             )))
                    )
-                  (((game-option-type normal-inverted))
+                  (((game-option-type normal-flipped))
                    ;; etc
                    (set! option-str (string-format "~S: ~S" (lookup-text! *common-text* (-> options index name) #f)
                                                         (if (-> (the-as (pointer uint32) (-> options index value-to-modify)))
                                                             (lookup-text! *common-text* (game-text-id normal) #f)
-                                                            (lookup-text! *common-text* (game-text-id inverted) #f)
+                                                            (lookup-text! *common-text* (game-text-id flipped) #f)
                                                             )))
                    )
```

Comment:
Remove this entire case and add it to where all the other ones are (it's right above this, but I can't link it for some reason).

Reply:
It will change the unselected text to just be the option's text and doesn't add the FLIPPED/DEFAULT at the end.